AllReadyApp/Web-App/AllReady/Views/Campaign/Index.cshtml

```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href="~/css/campaignStyle.css" asp-file-version="true" />
 </environment>
 
-<div class="row search-banner">
+<div class="row search-banner"> <!-- About to remove this div tag for issue #422 -->

```

Comment:
Looks like you're on the right path. If you can double check the div class for use (perhaps a JS file is bound to this class?) then this should be okay to remove per the request.